### PR TITLE
Right Side Weather View

### PR DESCRIPTION
**Pull request by theroyalblah**
When you aren't on the weather tab, on the right side instead of the time you can now see a small snippet of the weather.
This is handled by the top-corner-controller which selects which view, time or weather, that you should see dependent on the tab.
Screen Shot 2019-07-15 at 3 36 31 PM

### CODE REVIEW

File: app/javascript/components/widgets/weather/hourly-temps.jsx [hunk 1]
```diff
@@ -67,12 +67,12 @@ const getHourlyWeather = gql`
   }
 `;
 
-const HourlyTemps = ({ weather }) => {
+const HourlyTemps = ({ weather, hours }) => {
   const { data: hourlyWeathers } = weather.hourly;
-
+  const numHours = parseInt(hours, 10);
```
Comment (cbarber) [inline app/javascript/components/widgets/weather/hourly-temps.jsx]: Why not integer?

File: app/javascript/components/time-hero.jsx [hunk 2]
```diff
@@ -8,8 +8,9 @@ import { spacing } from '@lib/theme';
 import { LoadingMessage, ErrorMessage } from '@messages/default-messages';
 
 const Wrapper = styled(SpacedRow)`
+  margin-top: ${spacing.m};
   align-items: flex-end;
-  margin-bottom: ${spacing.xxl};
+  margin-bottom: 26px;
```
Comment (cbarber) [inline app/javascript/components/time-hero.jsx]: Does this orphan spacing.xxl?

**Comment by aaddaamm** [comment]
@theroyalblah just a rebase off latest master here once the comments are addressed.

File: app/javascript/components/time-hero.jsx (hunk 2 above)
Comment (theroyalblah) [reply app/javascript/components/time-hero.jsx]: Nope, though it's only used in one other place -- fixed-content.jsx

**Pull request merged**